lib/Elastica/Bulk/ResponseSet.php

```diff
@@ -42,16 +42,13 @@ public function getBulkResponses()
      */
     public function getError()
     {
-        $error = '';
-
         foreach ($this->getBulkResponses() as $bulkResponse) {
             if ($bulkResponse->hasError()) {
-                $error = $bulkResponse->getError();
-                break;
+                return $bulkResponse->getError();
```

Comment:
I without enough experience in big projects like that - **so if want revert as before** (but read enough books).
In this case `before` - 1 additional variable and 1 additional initialization, `break` - is same as `goto` to end of loop, so i change `goto` and `return` to 1 operation `return`.